scripts/sct_process_segmentation.py

```diff
@@ -906,14 +906,17 @@ def label_vert(fname_seg, fname_label, verbose=1):
     """
     # Open labels
     im_disc = Image(fname_label)
+    # change the orientation to RPI because the function label_segmentation assume this orientation as input
```

Comment:
Is this still needed now that you reorient to RPI the input segmentation inside `label_segmentation()`?

Reply:
Yes I think that we need it here too because in the loop (3 lines below) we compute the z coordinate of the labels --> z axis needs to be Superior-to-Inferior axis,

Reply:
in that case i would modify the comment, which currently says "because the function label_segmentation assume this orientation as input".